### PR TITLE
Update turbo tutorial

### PR DESCRIPTION
**Pull request by dme65**
This makes a few changes to the TuRBO tutorial.
- Use a similar lengthscale prior as in the TuRBO paper as the SingleTaskGP default doesn't work well for high-dimensional problems.
- Bump the dimensionality of the Ackley function up from 10 to 20.
- Use Thompson sampling inside a `no_grad` context.
- Use Cholesky by default, both for fitting and sampling.

### CODE REVIEW

File: tutorials/turbo_1.ipynb [hunk 1]
```diff
@@ -246,8 +251,9 @@
     "        X_cand[mask] = pert[mask]\n",
     "\n",
     "        # Sample on the candidate points\n",
-    "        thompson_sampling = MaxPosteriorSampling(model=model, replacement=False)\n",
-    "        X_next = thompson_sampling(X_cand, num_samples=batch_size)\n",
+    "        with torch.no_grad():  # We don't need gradients when using TS\n",
+    "            thompson_sampling = MaxPosteriorSampling(model=model, replacement=False)\n",
```
Comment (Balandat) [inline tutorials/turbo_1.ipynb:255]: nit: no need to put the constructor into the `no_grad` context

File: tutorials/turbo_1.ipynb [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "\n",
     "In this tutorial, we show how to implement Trust Region Bayesian Optimization (TuRBO) [1] in a closed loop in BoTorch.\n",
     "\n",
-    "This implementation uses one trust region (TuRBO-1) and supports either parallel expected improvement (qEI) or Thompson sampling (TS). We optimize the $10D$ Ackley function on the domain $[-10, 15]^{10}$ and show that TuRBO-1 outperforms qEI as well as Sobol.\n",
+    "This implementation uses one trust region (TuRBO-1) and supports either parallel expected improvement (qEI) or Thompson sampling (TS). We optimize the $20D$ Ackley function on the domain $[-5, 10]^{20}$ and show that TuRBO-1 outperforms qEI as well as Sobol.\n",
```
Comment (Balandat) [inline tutorials/turbo_1.ipynb:11]: any particular reason to change the bounds of the domain here?

File: tutorials/turbo_1.ipynb [hunk 3]
```diff
@@ -569,7 +644,7 @@
     "plt.xlabel(\"Number of evaluations\", fontsize=18)\n",
     "plt.title(\"10D Ackley\", fontsize=24)\n",
```
Comment (Balandat) [inline tutorials/turbo_1.ipynb:645]:
```suggestion
    "plt.title(\"20D Ackley\", fontsize=24)\n",
```

File: tutorials/turbo_1.ipynb (hunk 2 above)
Comment (dme65) [reply tutorials/turbo_1.ipynb:11]: I made it consistent with the TuRBO paper since I was looking at the 200D version for #798.

**Review by Balandat — APPROVED** [review]

**Pull request closed without merging**